dts/arm/nxp/nxp_rt11xx.dtsi

```diff
@@ -1214,3 +1214,9 @@
 	 */
 	status = "disabled";
 };
+
+nxp_cam_i2c: &lpi2c6 {};
+
```

Comment:
This probably should be in the board level overlay file

Reply:
yes, in fact, this is not sure (I even don't have datasheet for rt1080), so it's better to put in in the board level file. Done